pkg/webservice/handlers.go

```diff
@@ -98,6 +107,20 @@ func getApplicationsInfo(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func validateQueue(queueName string) error {
+	if queueName != "" {
+		queueNameArr := strings.Split(queueName, ".")
+		for _, name := range queueNameArr {
+			if !configs.QueueNameRegExp.MatchString(name) {
+				return fmt.Errorf("problem in queue query parameter parsing as queue param " +
+					"#{queueName} contains invalid queue name #{name}. Queue name must only have " +
```

Comment:
Not sure what happened here but this does not achieve what you want: use `%s` for the strings see [fmt.Errorf example](https://golang.org/pkg/fmt/#Errorf)

Reply:
Has been mis leaded with the way editor showing statements. Corrected now.